Makefile

```diff
@@ -195,9 +191,18 @@ root-coverage: ## generate coverage profiles for unit tests that require root
 		fi; \
 	done )
 
-coverage-integration: ## generate coverprofiles from the integration tests
-	@echo "$(WHALE) $@"
-	go test ${TESTFLAGS} -test.short -coverprofile="../../../${INTEGRATION_PACKAGE}/coverage.txt" -covermode=atomic ${INTEGRATION_PACKAGE} -test.root
+install-runc: ## install runc
```

Comment:
These are specific to the CI environment and shouldn't be makefile targets.

Reply:
Are they specific to CI? When I tried to run the integration suite locally I had to install these dependencies. It seems that anyone who is going to contribute to this project would need to install these. 

Is there a more appropriate place for this? Maybe a `Makefile.ci` or `Makefile.deps` ? In other projects these might be lines in a `Dockerfile`.

I'm happy to move it somewhere else.

Reply:
@dnephin While I appreciate that you are kind enough to install the dependencies, different environments have different ways of installing things. For example, on my local machine, I'd prefer to use `apt`. For newer packages, I might install to `/usr/local` or my home directory. Either way, this is up to the operator of the host. By putting these in the Makefile, you take this decision out of the operator's hand.

The main issue here is that when things are done in this automated fashion, the project resources expect these dependencies to be installed in a certain way. Over time, this creates a dependency on the installation methodology, which should be generic. Typically, with these assumptions, the project becomes less flexible and hard to port to other environments.

The reason that travis is the right place for these is that this is how the travis environment is setup. If we had a Dockerfile (which we could, at this point), it would have some other installation methodology, such as apk, to get these resources.

In other words, these were already in the right place.

Reply:
> By putting these in the Makefile, you take this decision out of the operator's hand.

I would argue that would only be true if project targets depended on the dependency install targets. As long as they are left optional, the operator can still make the decision to use the target, or do something else. For anyone using a docker image or a dedicated VM they would probably be fine with the more convenient option.

> when things are done in this automated fashion, the project resources expect these dependencies to be installed in a certain way.

I agree that the project having a dependency on specific paths is bad, but I don't agree that providing a convenient way of installing dependencies automatically leads to this dependency. As long as the project continues to use $PATH to find dependencies, it should be fine.

> If we had a Dockerfile (which we could, at this point)

That would be a good solution.

Since this change is not at all related to the primary change in the PR I've removed the commit. The dependency installation is back in `.travis.yml`.